cmat/output_generation/clinvar_to_evidence_strings.py

```diff
@@ -77,6 +78,15 @@ def clinvar_to_evidence_strings(string_to_efo_mappings, variant_to_gene_mappings
 
         # Catch any exceptions for this record so we can continue processing.
         try:
+            # Failure mode 0 (skip). Contains multiple clinical classification annotations.
+            # This is new as of V2 of the ClinVar XSD and should definitely be supported at some point,
+            # but as it can cause parsing complications we catch these cases first.
+            # See GH issue for context: https://github.com/EBIvariation/CMAT/issues/396
+            if len(clinvar_record.clinical_classifications) > 1:
+                logger.warning(f'Found multiple clinical classifications in record {clinvar_record.accession}')
+                report.clinvar_skip_multiple_clinical_classifications += 1
+                continue
```

Comment:
There's maybe another failure mode we're missing, which is if there's a single clinical classification but it's a string that isn't in [this enum](https://github.com/opentargets/json_schema/blob/master/schemas/disease_target_evidence.json#L1684)... this would include values like "Tier I - Strong" and others listed [here](https://github.com/EBIvariation/CMAT/issues/396#issuecomment-1898804129).

At it stands this is only caught when the evidence string is generated and validated, because we don't maintain the list of acceptable values in this code base. I'm not sure if we should change this or just let these be counted as invalid and investigated.